src/tests/test_fitting.py

```diff
@@ -10,6 +11,7 @@ def test_cart_spherical_conversions():
     cardinal directions and if cart_to_spherical() and spherical_to_cart() are inverses
     if each other
     """
+    print(np.__version__)
```

Comment:
Do we want to remove this?

Reply:
yeah that seams like a good idea, I'll update it in the next commit